src/scss/constants.scss

```diff
@@ -163,7 +163,7 @@ $loading-page-icon-transform-small-opacity: 0.4;
 // @debt the following locations are using styled-components, so can't use our helper function yet
 //   pages/Admin/Room/Modal/RoomModal.styles.ts (z-index: 101)
 //   components/atoms/Legend/Legend.styles.ts (z-index: 5)
-$z-layer-navbar: 30;
+$z-layer-navbar: 31;
```

Comment:
Well, the main reason why `.announcement-container-message` (announcement message) hides the `.nav-schedule .nav-schedule-clicked` (the < close button) is about their z-index. 
Right now they are equal (30) and absolutely positioned announcement message is higher. 

I've checked `$z-layer-navbar` is used only for `.navbar.navbar_playa` and it'd be a direct fix for the schedule to be higher than an announcement.

Reply:
[OOS] We might need better resolution to just updating numbers. Something like:
 `$z-layer-navbar:  $z-layer-announcement + 1;`

BTW, how hard is it to just yank the code from `RoomModal.styles.ts` and put it in `RoomModal.scss`?

Reply:
[OOS] We should also determine if we're going to put all z-index constants flat in `constants.scss` or have those values part of a single enum `z-layers` for consistency.

The first option might be more conducive to adding relative heights like the above OOS note